pyat/at/lattice/lattice_object.py

```diff
@@ -192,18 +249,21 @@ def repfunc(pass_method):
             elif pass_method == 'auto':
                 def ff(elem):
                     if not elem.PassMethod.endswith('RadPass'):
-                        elem.PassMethod = ''.join((elem.PassMethod[:-4], 'RadPass'))
+                        elem.PassMethod = ''.join(
+                            (elem.PassMethod[:-4], 'RadPass'))
                         elem.Energy = self.energy
             else:
                 def ff(elem):
                     elem.PassMethod = pass_method
                     elem.Energy = self.energy
             return ff
 
```

Comment:
I think we should warn if the user switches the radiation on when it is already on, i.e.
```
if self._radiation:
    warn(AtWarning('Radiation already on, switching it on anyway.'))
```